j5/boards/sr/v4/power_board.py

```diff
@@ -0,0 +1,129 @@
+"""Classes for the SR v4 Power Board."""
+
+from enum import Enum
+from typing import TYPE_CHECKING, List, Union, cast
+
+from j5.backends import Backend, Environment
+from j5.boards import Board
+from j5.components import LED, BatterySensor, Button, Piezo, PowerOutput
+
+if TYPE_CHECKING:
+    from j5.components import (  # noqa: F401
+        Component,
+        ButtonInterface,
+        PowerOutputInterface,
+        PiezoInterface,
+        BatterySensorInterface,
+        LEDInterface,
+    )
+    from typing import Type  # noqa: F401
+
+
+class PowerOutputPosition(Enum):
+    """A mapping of name to number of the PowerBoard outputs."""
+
+    H0 = 0
+    H1 = 1
+    L0 = 2
+    L1 = 3
+    L2 = 4
+    L3 = 5
+
+
+PowerOutputGroupIndex = Union[int, PowerOutputPosition]
+
+
+class PowerOutputGroup:
+    """A group of PowerOutputs on the PowerBoard."""
+
+    def __init__(self, backend: Backend, board: Board):
+        self._backend = backend
+        self._board = board
+        self._outputs = [
+            PowerOutput(n, self._board, cast("PowerOutputInterface", self._backend))
+            for n in range(0, 6)
+        ]
+
+    def power_on(self) -> None:
+        """Set all outputs in the group on."""
+        for output in self._outputs:
+            output.is_enabled = False
```

Comment:
Am I misunderstanding the `is_enabled` flag? In my mind 'turning on' the output would set `is_enabled` to true.

Reply:
This is in fact a typo

Reply:
Fixed in a460a75